src/redux/api.ts

```
@@ -0,0 +1,37 @@
+    if (error.response && error.response.status === 401) {
+      if (retry) {
+        retry = false;
+        return Promise.reject(error);
+      }
+
+      retry = true;
+
+      try {
+        await apiInstance.post("/auth/refresh-token");
+        return apiInstance(originalRequest);
+      } catch (e: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)